core/src/main/scala/codesearch/core/regex/lexer/RowPicker.scala

```diff
@@ -0,0 +1,14 @@
+package codesearch.core.regex.lexer
+
+import codesearch.core.regex.lexer.tokens.Token
+
+object RowPicker {
+
+  /**
+    * String builder from tokens
```

Comment:
This comment doesn't give any useful information that wasn't in the function name already.

The reason I'm asking for comments isn't because I want some stupid objective like "we have 100% comment coverage" or whatever -- that is useless. I want people to write comments so that the next person who joins our team will be able to understand what an unfamiliar piece of code does without having to ask around.

So, when you're writing a comment, you can imagine having to explain what this code does (not *how* but *what*) to a person who's not familiar with it.

Reply:
Sorry, 
I do not know how best to describe this feature. For example, "string picker from sequence Tokens"...

Reply:
You can ask somebody else who's not familiar with the code (e.g. @Kabowyad) to help you 🙂